### PR TITLE
docs: update teleport windows auth exe example

### PR DESCRIPTION
**Pull request by stevenGravy**
Updates example to use the downloaded exe name with version.

### CODE REVIEW

File: docs/pages/enroll-resources/desktop-access/getting-started.mdx [hunk 1]
```diff
@@ -84,10 +84,23 @@ interactively and select the Teleport certificate that you exported when prompte
 If you want to automate the installation process, you can run the setup program from
 an administrative Command Prompt or PowerShell console with the following command:
 
+<Tabs>
```
Comment (zmb3) [inline docs/pages/enroll-resources/desktop-access/getting-started.mdx:87]: We don't need tabs for this. Just use `teleport.version` unconditionally.

Comment (stevenGravy) [reply docs/pages/enroll-resources/desktop-access/getting-started.mdx:87]: The version are different though depending on which flow they're following. If they pull from community/ent that's 16.4 in this case. Cloud's exec name would be 16.3 version.


image

Comment (stevenGravy) [reply docs/pages/enroll-resources/desktop-access/getting-started.mdx:87]: one uses `teleport.version` and the other tab `cloud.version`.

Comment (stevenGravy) [reply docs/pages/enroll-resources/desktop-access/getting-started.mdx:87]: Did you want to take out the above tabs too?

Comment (stevenGravy) [reply docs/pages/enroll-resources/desktop-access/getting-started.mdx]: got it, please see update, tabs in both sections removed and only using `teleport.version`.

**Review by zmb3 — APPROVED** [review]

**Review by ptgott — APPROVED** [review]

**Pull request merged**